.github/workflows/nightly-ci-gpu.yml

```diff
@@ -0,0 +1,54 @@
+name: Nightly JAX CI on NVIDIA GPUs
```

Comment:
rename this file to nightly-ci-multiprocess-gpu.yml?

Reply:
Is it okay if we keep it generic now? Or do we need to keep it specific now and make it generic later?

Reply:
I would prefer that (keep specific then generic). I feel that we should name the file as to what its doing. If it does more things we can always rename

Reply:
Okay, renamed the file now.